index.html

```diff
@@ -208,6 +208,31 @@ <h3>GetUserMediaSemantics enum</h3>
         </table>
       </div>
     </section>
+    <section id="mediastreamconstraints-dictionary-extensions">
+      <h3>MediaTrackConstraintSet dictionary extensions</h3>
+      <div>
+        <pre class="idl"
+>partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean efficientPixelFormat;
+};</pre>
+        <section>
+          <h2>Dictionary {{MediaTrackConstraintSet}} Members</h2>
+          <dl data-link-for="MediaTrackConstraintSet" data-dfn-for=
+          "MediaTrackConstraintSet" class="dictionary-members">
+            <dt><dfn><code>efficientPixelFormat</code></dfn> of type <span class=
+            "idlMemberType">{{ConstrainBoolean}}</span></dt>
+            <dd>
+              <p>It is common for cameras to support different pixel formats for
```

Comment:
Maybe talk less of pixel formats? Something like
"Some capture modes are more expensive than others when it comes to power consumption. The user agent SHOULD label modes that incur significant power penalty as not power efficient. What format is or is not considered power efficient is up to the user agent, but modes that cause software MJPEG decode is an example of expensive modes."